api.py

```diff
@@ -693,3 +693,18 @@ def remove_asset(**kwargs):
 
 def _remove_asset(asset_id, **kwargs):
     return make_post_request({}, 'userassets/delete/{}'.format(asset_id), **kwargs)
+
+
+def account_exists(email, **kwargs):
+    try:
+        make_post_request({'email': email}, 'user/public/email-available', **kwargs)
```

Comment:
Something feels weird with this approach.  I'm assuming you're using post request to hide email address from a sniffer.  Now the URL states 'email-available' and returns a conflict 409 error.  Isn't 404 not found more appropriate here?  Which answers the question for URL 'email-available', success means yes, exception means no.